omod-2.2/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs2_2/DiagnosisResource2_2.java

```diff
@@ -19,19 +19,15 @@
 import org.openmrs.Diagnosis;
 import org.openmrs.api.context.Context;
 import org.openmrs.module.webservices.docs.swagger.core.property.EnumProperty;
-import org.openmrs.module.webservices.rest.SimpleObject;
 import org.openmrs.module.webservices.rest.web.RequestContext;
 import org.openmrs.module.webservices.rest.web.RestConstants;
 import org.openmrs.module.webservices.rest.web.annotation.PropertyGetter;
 import org.openmrs.module.webservices.rest.web.annotation.Resource;
 import org.openmrs.module.webservices.rest.web.representation.DefaultRepresentation;
 import org.openmrs.module.webservices.rest.web.representation.FullRepresentation;
 import org.openmrs.module.webservices.rest.web.representation.RefRepresentation;
-import org.openmrs.module.webservices.rest.web.representation.Representation;
```

Comment:
Did you run mvn clean package before doing the PR?

Reply:
@sacull I did and most tests passed except one at openmrs2_2 failed. I could not tell why exactly it failed

Reply:
So why did you do a PR when you know the code is corrupt because not all the tests pass?

Reply:
Am sorry sir but am working out to fix the exception

Reply:
Have addressed the comment

> It looks good to me now. ;)
> 
> But please remember not to do PR if you don't get tested locally, because after merge bugs may be introduced into OpenMRS code that will damage OpenMRS.

Its awesome to hear that the code looks pretty good. Have grasped the PR rules as per say.